health/micro-ui-react19/web/packages/css/src/pages/employee/healthdss.scss

```
@@ -1658,6 +1756,34 @@
+
+  &__custom-field {
+    display: flex;
+    flex-direction: column;
+    gap: 0.25rem;
+    width: 100%;
+    max-width: 27.5rem;
+
+    label {
+      font-family: Roboto;
+      font-weight: 400;
+      font-style: Regular;
```

Stylelint (17.14.0)

[error] Expected "Regular" to be "regular" (value-keyword-case)

(value-keyword-case)